modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/modify/ServiceModificationAuditDetails.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.models.service.modify;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.OffsetDateTimeSerializer;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+import org.eclipse.xpanse.modules.models.service.deploy.DeployRequest;
+import org.eclipse.xpanse.modules.models.service.deploy.DeployResource;
+import org.eclipse.xpanse.modules.models.service.enums.TaskStatus;
+
+/**
+ * Define view object for detail of the service modification audit.
+ */
+@Data
+public class ServiceModificationAuditDetails {
+
+
+    @NotNull
+    @Schema(description = "The id of the service modification audit.")
```

Comment:
Is it same as the modification request id?
Then we can add description as - The Id of the modification request.